### ISSUE TITLE
Extend flow/validate with meters validation

### ISSUE THREAD

**Issue by pzakatov** [telstra/open-kilda#1249]
**Scope**
1. Extend validate flow with meters validation on ingress rules
2. Extend switch rules validation with new meters focused sections 

**Details**
there is an OF message:

```
7.3.5.13 Meter Configuration
The OFPMP_METER_CONFIG multipart request message provides configuration for one or more meters.
The request body consists of a meter_id field, which can be set to OFPM_ALL to refer to all meters on
the switch.
104 © 2015; The Open Networking Foundation
OpenFlow Switch Specification Version 1.3.5
/ Body of OFPMP_METER and OFPMP_METER_CONFIG requests. /
struct ofp_meter_multipart_request {
uint32_t meter_id; / Meter instance, or OFPM_ALL. /
uint8_t pad[4]; / Align to 64 bits. /
};
OFP_ASSERT(sizeof(struct ofp_meter_multipart_request) == 8);
If the meter_id field does not exist on the switch then the switch must return an ofp_error_msg with
OFPET_METER_MOD_FAILED type and OFPGMFC_UNKNOWN_METER code.
The body of the reply consists of an array of the following structure:
/ Body of reply to OFPMP_METER_CONFIG request. Meter configuration. /
struct ofp_meter_config {
uint16_t length; / Length of this entry. /
uint16_t flags; / All OFPMF_ that apply. */
uint32_t meter_id; / Meter instance. /
struct ofp_meter_band_header bands[0]; /* The bands length is
inferred from the length field. */
};
OFP_ASSERT(sizeof(struct ofp_meter_config) == 8);
The fields are the same fields used for configuring the meter (see 7.3.4.4).
```

https://3vf60mmveq1g8vzn48q2o71a-wpengine.netdna-ssl.com/wp-content/uploads/2014/10/openflow-switch-v1.3.5.pdf

**Proposal**
`flow/validate` and `switch/validate/rules` must have the same output:
```
[
  {
    "switch": "switchID",
    "rules": {
      "missing": [],
      "proper": [],
      "misconfigured": [],
      "excess": []
    },
    "meters": {
      "missing": [],
      "proper": [],
      "misconfigured": [],
      "excess": []
    }
  }
]
```
In case of switch validation response will contain array with one requested switch unit. In case of flow validation the response will be with several switches affected by the flow.

Also we need to think about API versioning or change API path.

Related issues:
- #1022 FL returns 100 meters only
- #988 get all meters from NB
- #678 Meters support in ATDD
- #1593 Add ability to get all meters from Floodlight via Kafka

**Comment by nikitamarchenko** [comment]
FL endpoint /wm/kilda/meters/switch_id/{sw_id}

**Comment by Settis** [comment]
Description was updated with Proposal.

**Comment by dpoltavets** [comment]
Since this is already implemented for switch validation (#2066), we need to add meter validation only to flow validation.
